Makefile

```diff
@@ -222,7 +223,10 @@ ifeq ($(USE_GPERFTOOLS), 1)
 		ifeq (,$(FIND_LIBFILE))
 			FIND_LIBFILE=$(wildcard /usr/local/lib/lib$(FIND_LIBNAME).$(FIND_LIBFILEEXT))
 			ifeq (,$(FIND_LIBFILE))
-				USE_GPERFTOOLS=0
+				FIND_LIBFILE=$(wildcard /usr/lib64/lib$(FIND_LIBNAME).$(FIND_LIBFILEEXT))
+				ifeq (,$(FIND_LIBFILE))
+					USE_GPERFTOOLS=0
```

Comment:
Please print a message stating that GPERFTOOLS has been disabled

Reply:
But what you request had already been implemented. Please refer
https://github.com/apache/incubator-mxnet/blob/2dbd143e4892bb9ad4aa1835c79f0046603e3531/Makefile#L264